ansible/roles/drupal/tasks/main.yml

```diff
@@ -204,15 +204,8 @@
   args: 
     chdir: "{{ drupal8_root }}/web"
 
-### Extra modules ###
```

Comment:
Does moving the extra modules installation after custom modules have any side effects?